gdx/src/com/badlogic/gdx/graphics/g2d/PixmapPacker.java

```diff
@@ -101,6 +101,7 @@
 	Format pageFormat;
 	int padding;
 	boolean duplicateBorder;
+	Color currentTransparentColor = new Color(0f, 0f, 0f, 0f);

```

Comment:
- ```getTransparentColor``` means that it belongs to the ```PixmapPacker```.
- ```getCurrentTransparentColor``` means that it belongs to the current state of the ```PixmapPacker```, you can vary it between glyphs/pages/etc for your reasons.